frontend/server/src/main/java/com/amazonaws/ml/mms/util/messages/ModelLoadModelRequest.java

```diff
@@ -27,15 +27,15 @@
     private String gpu;
 
     public ModelLoadModelRequest() {
-        super("load", null);
+        super(CommandStrings.LOAD.getCommand(), null);
```

Comment:
It's better use CommandStrings type in Request class as well.

Reply:
Done